examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,558 @@
+"""
```

Comment:
@arokem I believe that you mentioned in passing that this example might be relevant to baby-AFQ wondering if you could elaborate and whether we would want to share with Mareike

Reply:
OK - this is now fleshed out more fully in #487